.idea/misc.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we don't need to commit .idea files to git.
Please remove them and read about .gitignore file: http://git-scm.com/docs/gitignore